cpp/ps0004/ps0004_solution.cpp

```diff
@@ -2,39 +2,32 @@
 using namespace std;
 
 int main() 
-
 {
-    int input;
-
-
+    // prompting user for number 1/1.5
+    float input;
     cout << "Enter a number: ";
+    cin >> input;

```

Comment:
- [x] declared a variable to store the user input
- [ ] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input